pallets/Polkadex_Ido/src/test.rs

```diff
@@ -755,4 +755,34 @@ fn test_vote_for_round_no_vote_majority() {
             false
         );
     });
+}
+
+#[test]
+fn test_get_reserve_amount() {
+    let balance: Balance = 100;
+    let open_block_number = 11;
+    let closing_block_number = 50;
+    let cid = [0_u8;32];
+    ExtBuilder::default().build().execute_with(|| {
+        assert_eq!(
+            PolkadexIdo::register_round(
+                Origin::signed(ALICE),
+                cid,
+                AssetId::CHAINSAFE(H160::from_low_u64_be(24)),
+                balance,
+                AssetId::POLKADEX,
+                balance,
+                open_block_number,
+                balance,
+                balance,
+                balance,
+                balance,
+                closing_block_number
+            ),
+            Ok(())
+        );
```

Comment:
This test only tests if register_round returns ok. Name indicate it should test get reserve amount

Reply:
https://github.com/Polkadex-Substrate/Polkadex/blob/a09408a6d98d9cfce5d5a9984eba4bfba3b6c49b/pallets/Polkadex_Ido/src/test.rs#L767

https://github.com/Polkadex-Substrate/Polkadex/blob/a09408a6d98d9cfce5d5a9984eba4bfba3b6c49b/pallets/Polkadex_Ido/src/test.rs#L796

https://github.com/Polkadex-Substrate/Polkadex/blob/a09408a6d98d9cfce5d5a9984eba4bfba3b6c49b/pallets/Polkadex_Ido/src/test.rs#L799